packages/model-viewer/rollup.config.js

```diff
@@ -30,48 +30,38 @@ const onwarn = (warning, warn) => {
   }
 };
 
-let plugins = [resolve({ dedupe: 'three' }), replace({ 'Reflect.decorate': 'undefined' })];
+let commonPlugins = [
+  resolve({ dedupe: 'three' }),
+  replace({ 'Reflect.decorate': 'undefined' })
+];
 
 const watchFiles = ['lib/**'];
 
-const outputOptions = [
-  {
-    input: './lib/model-viewer.js',
-    output: {
-      file: './dist/model-viewer.js',
-      sourcemap: true,
-      format: 'esm',
-      name: 'ModelViewerElement',
-    },
-    watch: {
-      include: watchFiles,
-    },
-    plugins,
-    onwarn,
+const createModelViewerOutput = (file, format, globals = {}, plugins = commonPlugins, external = []) => ({
```

Comment:
It seems like `globals` and `externals` are related for our uses - should we just take `external` and populate `global` from that?

Reply:
Good point, updated it, thanks for the review!